bridge/contracts/libraries/errors/DutchAuctionErrors.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity ^0.8.11;
+
+library DutchAuctionErrors {
+    error IcorrectInitialPrices();
```

Comment:
```suggestion
    error IncorrectInitialPrices();
```

Reply:
Thanks @diligentis. Finally we are not raising any error so file was deleted